pkg/skippeers/skippeers.go

```diff
@@ -70,4 +72,15 @@ func (l *List) PruneExpired() {
 			delete(l.skip, k)
 		}
 	}
+
+	return count
+}
+
+func (l *List) Reset() {
```

Comment:
@aloknerurkar @vladopajic @mrekucci just to confirm, do we need this Reset to prevent memory leak?

Reply:
I dont think we need it as this skip list is local to the function and once the function returns it should be GCd.

Reply:
i was not sure since it's a map within a map